src/mechanics/cell/SimpleAssember.h

```diff
@@ -11,58 +11,58 @@ namespace NuTo
 class SimpleAssembler
 {
 public:
-    SimpleAssembler(const NuTo::DofContainer<int>& numIndependentDofs, const NuTo::DofContainer<int>& numDependentDofs)
+    SimpleAssembler(NuTo::DofContainer<int> numIndependentDofs, NuTo::DofContainer<int> numDependentDofs)
         : mNumIndependentDofs(numIndependentDofs)
         , mNumDependentDofs(numDependentDofs)
     {
     }
 
-    GlobalDofVector BuildVector(const Groups::Group<NuTo::CellInterface>& cells,
-                                const std::vector<NuTo::DofType*>& dofTypes, const VectorOperation& op) const
+    GlobalDofVector BuildVector(const Groups::Group<NuTo::CellInterface>& cells, Groups::Group<NuTo::DofType> dofTypes,
```

Comment:
What advantage has using group over `std::vector` in this case?

Reply:
Value semantics. Avoidance of (accidentally?) duplicate dof types may also be helpful. But mainly value semantics. We could also have that with `std::vector<std::reference_wrapper<DofType>>`. `Group<DofType>` is much shorter.

Reply:
In other places you're moving towards using `DofType` as a value, so why not here?

Reply:
Right, I haven't thought of that. If you prefer `std::vector` over `NuTo::Group`, I will change that.

Reply:
I have no strong allegiance to either; I was just curious.

Reply:
You decide what to do!

Reply:
Is that so? 😬 Then I decree `std::vector`.